halotools/utils/tests/test_rotate_vector_collection.py

```diff
@@ -0,0 +1,55 @@
+"""
+"""
+import numpy as np
+from astropy.utils.misc import NumpyRNGContext
+
+from rotations.rotate_vector_collection import rotate_vector_collection
+from rotations.rotations3d import *
+
+__all__ = ('test_rotation_3d',
+           )
+
+fixed_seed = 43
+
+def test_rotation_1():
+    """
+    single rotation matrix + single set of points
+    """
+    
+    # create a single rotation matrix
+    nsets = 1
+    ndim = 3
+    v1 = np.random.random((nsets,ndim))
+    v2 = np.random.random((nsets,ndim))
+    rot_m = rotation_matrices_from_vectors(v1,v2)
+    rot = rot_m[0]
+
+    # create a single set of vectors
+    npts = 1000
+    ndim = 3
+    v3 = np.random.random((npts,ndim))
+
+    v4 = rotate_vector_collection(rot, v3)
+
+    assert np.shape(v4)==(npts, ndim)
```

Comment:
Is there a unit test elsewhere that enforces correct behavior of rotate_vector_collection? I think a non-trivial test would just be forcing that the rotated vector is indeed v2